lib/dolly/search_index.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'forwardable'
+
+module Dolly
+  class SearchIndex
+    class << self
+      extend Forwardable
+
+      ALL_DOCS = '_all_docs'
+      DESIGN = '_search'
+      ROWS_KEY = :rows
+      DESIGN_PREFIX = '_design/'
+
+      def_delegators :connection, :get, :post
+
+      def all
+        get(DESIGN)[:indexes]
+      end
+
+      def create(name, fields)
```

Comment:
IMO these actions should always receive a db name. Do we want to create global search indexes?

Reply:
ah! this file is no used actually... let me removed